src/model/item-list.model.ts

```diff
@@ -0,0 +1,15 @@
+
+export class ItemListModel{
```

Comment:
Why do you need a class?
I would imagine
```
interface ItemListModel {
  id: string;
  value: string;
  done: boolean;
}
```
should be enough.

Reply:
When I think that a model may change in the future and it may need a class method I prefer to implement it as a class at the outset, and I often add the method that returns a default object that may be useful in some cases. Usually at work we always did it this way